src/Detector.php

```diff
@@ -64,6 +64,21 @@ public function determineMimeType(\SplFileObject $file): string
         return 'text/plain; charset=utf-8';
     }
 
+    /**
+     * Read width amd height from a image file
+     *
+     * @param string $filePath
```

Comment:
As this function isn't `void` (no return value), would you please include the return type in the DocBlock, such as:

```
@return array [width, height]
```